libc/src/__support/FPUtil/BasicOperations.h

```diff
@@ -247,6 +247,14 @@ LIBC_INLINE T fdim(T x, T y) {
   return (x > y ? x - y : 0);
 }
 
+template <typename T, cpp::enable_if_t<cpp::is_floating_point_v<T>, int> = 0>
+LIBC_INLINE int issignaling(const T &x) {
+  FPBits<T> sx(x);
+  if (LIBC_UNLIKELY(sx.is_signaling_nan()))
```

Comment:
just `return sx.is_signaling_nan();` should be enough.  `bool -> 0/1 int` is well-defined in C and C++.

Reply:
Right. Done.